docs/source/workflow_outputs.rst

```diff
@@ -321,9 +321,11 @@ Annual unmet hours are listed below.
   =========================  =====
   Unmet Hours: Heating (hr)  Number of hours where the heating setpoint is not maintained.
   Unmet Hours: Cooling (hr)  Number of hours where the cooling setpoint is not maintained.
+  Unmet Hours: Driving (hr)  Number of hours where the driving schedule of electric vehicles is not met
   =========================  =====
 
-These numbers reflect the number of hours during the heating/cooling season when the conditioned space temperature deviates more than 0.2 deg-C (0.36 deg-F) from the heating/cooling setpoint.
+The unmet heating and cooling numbers reflect the number of hours during the heating/cooling season when the conditioned space temperature deviates more than 0.2 deg-C (0.36 deg-F) from the heating/cooling setpoint.
+The unmet driving numbers include any time in which the discharge schedule is exceeds 0 while the EV battery's state of charge is at its minimum level.
```

Comment:
```suggestion
The unmet driving numbers include any time in which the discharge schedule exceeds 0 while the EV battery's state of charge is at its minimum level.
```

Reply:
I have to say that this output continues to confuse me, especially when you have situations where EV charging can occur away from the house.

At a minimum, I feel like we need some more description about what this means for the simulation results, because this is going to be a foreign concept to a lot of users. Is it correct to say that the EV charging end use is lower than it should be because some EV driving miles did not occur, and therefore the corresponding EV charging did not occur? Or something like this?

Reply:
Re-worked the description here: https://github.com/NREL/OpenStudio-HPXML/pull/1533/commits/b03246453b03bd749a51b1da94fd839cb7a2dbf8, let me know your thoughts